spec/statelint_spec.rb

```diff
@@ -293,4 +293,11 @@
     expect(problems.size).to eq(1)
     expect(problems[0]).to include('may have only one of ["HeartbeatSeconds", "HeartbeatSecondsPath"]')
   end
+
+  it 'should allow a Comment field in Catcher' do
+    j = File.read "test/succeed-with-comment-in-catcher.json"
```

Comment:
You will need to create this file, then execute `bundle exec rspec` in the repo's root directory to run the tests. This currently fails because the definition file referenced here does not exist

```
$ bundle exec rspec
.........................F

Failures:

  1) StateMachineLint should allow a Comment field in Catcher
     Failure/Error: j = File.read "test/succeed-with-comment-in-catcher.json"

     Errno::ENOENT:
       No such file or directory @ rb_sysopen - test/succeed-with-comment-in-catcher.json
     # ./spec/statelint_spec.rb:298:in `read'
     # ./spec/statelint_spec.rb:298:in `block (2 levels) in <top (required)>'

Finished in 1.01 seconds (files took 0.06777 seconds to load)
26 examples, 1 failure

Failed examples:

rspec ./spec/statelint_spec.rb:297 # StateMachineLint should allow a Comment field in Catcher
```

Reply:
Whoops, forgot to commit that file 😬 . Pushed 🙂 

```
root@6e7c8b1f3fbd:/opt/code# bundle install
Bundler 2.4.10 is running, but your lockfile was generated with 2.3.7. Installing Bundler 2.3.7 and restarting using that version.
Fetching gem metadata from https://rubygems.org/.
Fetching bundler 2.3.7
Installing bundler 2.3.7
Fetching gem metadata from https://rubygems.org/.
Using rake 13.0.6
Using bundler 2.3.7
Using diff-lcs 1.5.0
Installing json 2.6.2 with native extensions
Installing rspec-support 3.11.0
Installing rspec-core 3.11.0
Installing rspec-expectations 3.11.0
Installing rspec-mocks 3.11.1
Installing rspec 3.11.0
Installing j2119 0.4.0
Updating files in vendor/cache
Bundle complete! 3 Gemfile dependencies, 10 gems now installed.
Use `bundle info [gemname]` to see where a bundled gem is installed.
root@6e7c8b1f3fbd:/opt/code# bundle exec rspec
..........................

Finished in 1.08 seconds (files took 0.06443 seconds to load)
26 examples, 0 failures
```